fontforge/bezctx_ff.c

```diff
@@ -1,28 +1,52 @@
-/* This file written by George Williams to provide a gateway to fontforge */
-/* it it a modification of Raph's bezctx_ps.c */
+/*
+Copyright: 2007 Raph Levien
+License: GPL-2+
+    bezctx_ps.c - ppedit - A pattern plate editor for Spiro splines.
+Copyright: 2007 George Williams
+    bezctx_ff.c - Provide a gateway into FontForge with added modification.
+Contributions: Yuri Chornoivan, Jonathan Hanna, Joe Da Silva, Reuben Thomas,
+    Franke-Trampe
```

Comment:
On July 19, 2016 02:42:12 PM Frank Trampe wrote:
> Name is wrong.

Sorry about that, should have double checked - my error.